libass/ass_utils.c

```diff
@@ -271,3 +271,13 @@ int ass_lookup_style(ASS_Track *track, char *name)
             track, name, track->styles[i].Name);
     return i;
 }
+
+void *ass_malloc(size_t size)
```

Comment:
nit: didn’t notice before, but i think this fits better into `ass_library.c`. We have several other public API functions there which don't belong to a more specific file, while `ass_utils` is more for internal utilities

Reply:
Agreed. I guess the counterpoint is that unlike other public API functions, this one is actually used internally throughout the library?

Reply:
We just use `(m|c)alloc`  and `free` directly internally; not `ass_*`